google/earthenterprise
# #1593: Fix Assorted Python Dependency Issues

Updates to fix various python dependency issues:

- Portable shebang fixed and documentation changed.
- Pyyaml now installs to the correct location.
- OpenGEE rpms now require python27-lxml (to ensure that it gets installed automatically).

To test:

- Build portable on el6 and verify that it builds with and works with python 2.7.
- Build and install on el6 and verify that glc_assembler works.
- Take built RPMs and install on a different el6 machine. Verify that cutter works.

These changes were made in conjunction with documentation changes on the [wiki](https://github.com/google/earthenterprise/wiki/Portable-Server#run-time-prerequisites).
